recipes/nss_wrapper/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+mkdir build
+cd build
+cmake -LAH .. \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"
```

Comment:
Should we add the build type to just avoid any problem? :grimacing: 
```suggestion
    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
    -DCMAKE_BUILD_TYPE=Release
```